scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -330,13 +320,14 @@ def parse_args(o3de_context: exp.O3DEScriptExportContext):
                                            disable_override_desc="Skip the building of the O3DE toolchain executables (AssetBundlerBatch, AssetProcessorBatch) as part of the export process. "
                                                                  "The tools must be already available at the location specified by the '--tools-build-path' argument.")
 
-        default_build_tools_path = export_config.get_value(exp.SETTINGS_DEFAULT_BUILD_TOOLS_PATH.key, exp.SETTINGS_DEFAULT_BUILD_TOOLS_PATH.default)
-        parser.add_argument('-tbp', '--tools-build-path', type=pathlib.Path, default=pathlib.Path(default_build_tools_path),
-                            help=f'Designates where the build files for the O3DE toolchain are generated. If not specified, default is {default_build_tools_path}')
+        
+        # Do not supply a default path for the tools build path. This should be inferred by the export script based on engine installation.
+        parser.add_argument('-tbp', '--tools-build-path', type=pathlib.Path, default=None,
+                            help=f'Designates where the build files for the O3DE toolchain are generated. If not specified, the path is inferred based on engine installation.')
```

Comment:
```suggestion
                            help=f'Designates where the build files for the O3DE toolchain are generated. For installed/SDK engines, the path is inferred based on where the engine is installed. For engines built from source code, the default location is <project_path>/build/tools, unless overwritten by this parameter. You may want to specify -tbp to save time recompiling tools under <project_path>/build/tools, if you've already built tools locally. For example, under <project_path>/build/windows')
```

Reply:
it's not always project folder. For engine_centric, it might refocus around the engine_folder. I left this somewhat generic to prervent such confusion